model/chain.go

```diff
@@ -0,0 +1,35 @@
+package model
+
+import "fmt"
+
+// Enum for Chain
+type Chain struct {
+	Name            string
+	NativeAsset     *AssetInfo
+	AddressMappings map[*AssetInfo]*AssetInfo // maps from a fixed set of assets from the remote chain to another fixed set of assets on the native chain (for now hard-coded, later on load from db)
+}
+
+var (
+	ChainStellar *Chain = &Chain{Name: "Stellar", NativeAsset: AssetStellar_XLM, AddressMappings: map[*AssetInfo]*AssetInfo{
```

Comment:
this definition is critical.
It defines everything that we have available on a given chain such as the native asset and all assets we support and how to convert from remote chains.
This format will be updated as we add more code so this model of a Chain is not finalized and is in-flux